### ISSUE TITLE
No local logins? I don't plan on getting an o-auth provider account anywhere will there ever be an account management for local users?

### ISSUE THREAD

**Issue by Leopere** [Eugeny/tabby-web#116]
I have no interest in using any of the providers you have for my local setup for any reason is there a version of the world where there will be basic auth for tabby web?

**Comment by Eugeny** [comment]
There's an alternative implementation of the config sync API that doesn't uses token auth: https://github.com/Clem-Fern/rtabby-web-api

**Comment by Leopere** [comment]
> There's an alternative implementation of the config sync API that doesn't uses token auth: https://github.com/Clem-Fern/rtabby-web-api
that was super fast thanks for that reply! I'm kinda bummed its an unofficial alternative but grateful it exists is there a way to bake your web frontend with this non token based backend?

**Comment by Eugeny** [comment]
No, it's completely unrelated. If you can come up with a PR adding passowrd auth, I'm happy to merge it.

**Comment by smashah** [comment]
status of the PRs?
